.github/workflows/dockerfiles.yaml

```diff
@@ -0,0 +1,30 @@
+name: Update Dockerfiles on Default Branch Merge
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
Is there an option for merged only?

Reply:
yes, good catch. there's a `merged` key in the webhook payload that I'll need to check for that.